platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/io/OutputStreamFactory.java

```diff
@@ -17,10 +17,9 @@
 package com.swirlds.logging.io;
 
 import static com.swirlds.logging.api.extensions.handler.LogHandler.PROPERTY_HANDLER;
-import static com.swirlds.logging.utils.ConfigUtils.configValueOrElse;
-import static com.swirlds.logging.utils.ConfigUtils.readDataSizeInBytes;
```

Comment:
This is a limitation of the `javadoc` tool. It complains about importing deprecated methods directly and there is no way to suppress that on an import. Hence, this is replaced by a normal import.

Reply:
lol